pkg/monitor/api/types.go

```diff
@@ -22,6 +22,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/cilium/cilium/pkg/identity"
```

Comment:
For reviewers: I'm unsure whether it's fine to import this package here.

Reply:
Using https://github.com/KyleBanks/depth :

```
$ depth ./pkg/identity
github.com/cilium/cilium/pkg/identity
  ├ errors
  ├ fmt
  ├ net
  ├ strconv
  ├ sync
  ├ github.com/cilium/cilium/pkg/k8s/apis/cilium.io
  ├ github.com/cilium/cilium/pkg/labels
    ├ bytes
    ├ crypto/sha512
    ├ encoding/json
    ├ fmt
    ├ net
    ├ sort
    ├ strings
    ├ github.com/cilium/cilium/pkg/logging/logfields
      └ fmt
    └ github.com/cilium/cilium/vendor/github.com/sirupsen/logrus
      ├ bufio
      ├ bytes
      ├ context
      ├ encoding/json
      ├ fmt
      ├ io
      ├ log
      ├ os
      ├ reflect
      ├ runtime
      ├ sort
      ├ strconv
      ├ strings
      ├ sync
      ├ sync/atomic
      ├ time
      ├ unicode/utf8
      └ github.com/cilium/cilium/vendor/golang.org/x/sys/unix
        ├ bytes
        ├ encoding/binary
        ├ math/bits
        ├ runtime
        ├ sort
        ├ strings
        ├ sync
        ├ syscall
        ├ time
        ├ unsafe
        └ github.com/cilium/cilium/vendor/golang.org/x/sys/internal/unsafeheader
          └ unsafe
  └ github.com/cilium/cilium/pkg/lock
    ├ context
    ├ sync
    ├ sync/atomic
    ├ github.com/cilium/cilium/vendor/github.com/sasha-s/go-deadlock
      ├ bufio
      ├ bytes
      ├ fmt
      ├ io
      ├ io/ioutil
      ├ os
      ├ os/user
      ├ path/filepath
      ├ runtime
      ├ strings
      ├ sync
      ├ time
      └ github.com/cilium/cilium/vendor/github.com/petermattis/goid
        ├ bytes
        ├ runtime
        └ strconv
    └ github.com/cilium/cilium/vendor/golang.org/x/sync/semaphore
      ├ container/list
      ├ context
      └ sync
38 dependencies (28 internal, 10 external, 0 testing).
```

Seems like a lot but I'm guessing the callers *probably* already pull in many of these?